master/buildbot/test/unit/www/test_oauth.py

```diff
@@ -157,6 +159,20 @@ def test_getGithubELoginURL(self):
                "scope=user%3Aemail+read%3Aorg")
         self.assertEqual(res, exp)
 
+    @defer.inlineCallbacks
+    def test_getGithubELoginURL_v4(self):
+        res = yield self.githubAuthEnt_v4.getLoginURL('http://redir')
+        exp = ("https://git.corp.fakecorp.com/login/oauth/authorize?client_id=ghclientID&"
```

Comment:
usually we use example.com for example urls

Reply:
So does that mean I need to use git.example.com instead of git.corp.fakecorp.com?

Reply:
Yes.

Reply:
Actually, in this case I see that the test was already using fakecorp domain before this PR, so I think that for consistency using it in subsequent test cases makes sense. Cleanup could happen in a separate unrelated PR, so you don't need to do anything.